testdata/scripts/extract.txt

```diff
@@ -597,47 +597,47 @@ curl -X POST \
+| branch            | string           | TODO: add comment.                          |
+| customer          | string           | TODO: add comment.                          |
+| debit_transaction | DebitTransaction | TODO: add comment.                          |
+| interest_rate     | string           | TODO: add comment.                          |
+| maturity_date     | string           | TODO: add comment.                          |
+| minor             | string           | TODO: add comment.                          |
 
 ##### Objects
 
 ###### AccountRole
 
-| Name         | Type       | Description        |
-|--------------|------------|--------------------|
-| EntityNumber | string     | TODO: add comment. |
-| EntityType   | EntityType | TODO: add comment. |
-| Role         | string     | TODO: add comment. |
+| Name          | Type       | Description        |
+|---------------|------------|--------------------|
+| entity_number | string     | TODO: add comment. |
+| entity_type   | EntityType | TODO: add comment. |
+| role          | string     | TODO: add comment. |
 
 ###### DebitTransaction
 
-| Name         | Type   | Description        |
-|--------------|--------|--------------------|
-| Amount       | string | TODO: add comment. |
-| DebitAccount | string | TODO: add comment. |
-| ExchangeRate | string | TODO: add comment. |
-| IsFx         | bool   | TODO: add comment. |
+| Name          | Type   | Description        |
+|---------------|--------|--------------------|
+| amount        | string | TODO: add comment. |
+| debit_account | string | TODO: add comment. |
+| exchange_rate | string | TODO: add comment. |
+| is_fx         | bool   | TODO: add comment. |
 
 ### Responses
 
 #### Response body
 
-| Name                    | Type   | Description                                                                |
-|-------------------------|--------|----------------------------------------------------------------------------|
-| AccountID               | string | AccountID is the unique identifier of the newly created account.           |
-| CreditTransactionNumber | string | CreditTransactionNumber is the transaction number of the credited account. |
-| DebitTransactionNumber  | string | DebitTransactionNumber is the transaction number of the debited account.   |
-| Minor                   | string | TODO: add comment.                                                         |
+| Name       | Type   | Description                                                                |
+|------------|--------|----------------------------------------------------------------------------|
+| account_id | string | AccountID is the unique identifier of the newly created account.           |
+| number     | string | CreditTransactionNumber is the transaction number of the credited account. |
+| number     | string | DebitTransactionNumber is the transaction number of the debited account.   |
```

Comment:
since there are two fields named `number`, while originally there is `CreditTransactionNumber` and `DebitTransactionNumber`. No idea where it goes from, probably unrelated to this PR though

Reply:
I tracked the issue to `pulpMd`; pulpMd is too dense to me to get what exactly is going on there, so I made an issue :)

https://github.com/Kunde21/pulpMd/issues/5